### ISSUE TITLE
The 512*512 model checkpiont?

### ISSUE THREAD

**Issue by FanqingM** [FoundationVision/VAR#32]
Hello, It is a nice work with very insteresting and intuitive idea.
It seems that there only has ckp with 256, I wonder would you like to share the ckp with 512?

**Comment by keyu-tian** [comment]
Yes we plan to upload this, just in these days. please stay tuned.

**Comment by Achazwl** [comment]
> Yes we plan to upload this, just in these days. please stay tuned.

Hello, what's the plan now
